src/Chapter1.hs

```diff
@@ -631,8 +643,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let npositive = if n < 0 then n * (-1) else n
+        first = mod npositive 10
+        second = mod (div npositive 10) 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs n) 10
div (abs n) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
I see, so is there a simpler way than:

```
sumLast2 :: Int -> Int
sumLast2 n =
    let npositive = abs (n)
        (a, x) = divMod npositive 10
        y = mod a 10
    in x + y
```